arelle/plugin/validate/NL/rules/nl_kvk.py

```diff
@@ -300,6 +301,64 @@ def rule_nl_kvk_3_2_3_1 (
         )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        DISCLOSURE_SYSTEM_NL_INLINE_2024
+    ],
+)
+def rule_nl_kvk_3_2_4_1 (
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    NL-KVK.3.2.4.1: Inconsistent numeric facts are prohibited.
+    """
+    problematicFacts= []
+    numericFacts = [fact for fact in val.modelXbrl.facts if fact is not None and fact.isNumeric]
+    if len(numericFacts) > 0:
+        for duplicateFactSet in getDuplicateFactSets(numericFacts, False):
+            if duplicateFactSet.areAnyInconsistent:
+                problematicFacts.append(duplicateFactSet)
+    if len(problematicFacts) > 0:
+        yield Validation.error(
+            codes='NL.NL-KVK.3.2.4.1.inconsistentDuplicateNumericFactInInlineXbrlDocument',
+            msg=_('Inconsistent numeric facts are prohibited.'),
+            modelObject = problematicFacts
```

Comment:
`problematicFacts` is a `list[DuplicateFactSet]` here. I don't think Arelle knows how to build refs from that. You may want to group all of the `duplicateFactSet.facts` into a single list or fire the error for each duplicate fact set similar to [this](https://github.com/Arelle/Arelle/blob/ac9434f43efd2591ad5b415f2faafa2abdfd8a4d/arelle/ValidateXbrl.py#L696).

Reply:
The conformance suite for 3.2.4.2 indicates that only one error per validation is expected so I went the list route.